popups.js

```diff
@@ -0,0 +1,20 @@
+const modalBtn = document.querySelectorAll('#modal-open');
+const closeModal = document.querySelector('#modal-close');
+const modalOverlay = document.querySelector('#modal-overlay');
+const modal = document.querySelector('.modal');
+
+function openModal() {
```

Comment:
- [ ] All the projects open the same popup with the same info, please consider fixing this so that when you click on the ```See Project``` button every project opens a unique popup with the proper information and image

![Screenshot_20220513_233022](https://user-images.githubusercontent.com/25227022/168397420-3ca71956-6189-4a59-8c4e-a65e09d110c1.png)